src/cocotb/types/array.py

```diff
@@ -19,6 +19,7 @@ class Array(ArrayLike[T]):
     The indexes of an array can start or end at any integer value, they are not limited to 0-based indexing.
     Indexing schemes can be either ascending or descending in value.
     An array's indexes are described using a :class:`~cocotb.types.Range` object.
+    Passing an :class:`int` for the *range* parameter acts as shorthand for ``Range(range-1, "downto", 0)``.
```

Comment:
```suggestion
    Passing an :class:`int` as second positional argument acts as shorthand for ``Range(range-1, "downto", 0)``.
```

Using wording like this would make it less tepmting to write `range=17`?